src/Installers/Windows/WindowsHostingBundle/1033/thm.wxl

```diff
@@ -52,9 +52,9 @@
 
   <!-- Custom UI strings -->
   <String Id="Welcome"><!--_locComment_text="{Locked='[WixBundleName]'}"-->Welcome to the [WixBundleName] Setup.</String>
-  <String Id="InstallResetIIS">Please restart IIS after the installation completes. You can find additional information &lt;a href="https://aka.ms/aspnet/8.0/host-and-deploy-with-iis"&gt;here&lt;/a&gt;.</String>
-  <String Id="InstallNoIIS">IIS is not enabled on this machine. If you intend to run ASP.NET Core applications with IIS, you must install IIS before running this installer. You can find additional information &lt;a href="https://aka.ms/aspnet/8.0/host-and-deploy-with-iis"&gt;here&lt;/a&gt;.</String>
+  <String Id="InstallResetIIS"><!--_locComment_text="'here' is to be translated"-->Please restart IIS after the installation completes. You can find additional information &lt;a href="https://aka.ms/aspnet/8.0/host-and-deploy-with-iis"&gt;here&lt;/a&gt;.</String>
```

Comment:
Translation of "here" was removed in 232ec2e3a48b0c1f60dd02617891ca84b87aa9bf. Again, that commit was unfortunately checked in.